src/sequence.jl

```diff
@@ -107,30 +107,30 @@ end
 Base.findfirst(seq::Sequence, val) = findnext(seq, val, 1)
 Base.findlast(seq::Sequence, val) = findprev(seq, val, endof(seq))
 
+
+# GC content
+# ----------
+
 """
     gc_content(seq::Sequence)
 
 Calculate GC content of `seq`.
 """
 function gc_content(seq::Sequence)
     if !(eltype(seq) <: NucleicAcid)
-        error("elements must be nucleotides")
-    end
-
-    gc = 0
-    for x in seq
-        if isGC(x)
-            gc += 1
-        end
+        throw(ArgumentError("not a nucleic acid sequence"))
     end
-
     if isempty(seq)
         return 0.0
     else
-        return gc / length(seq)
+        return count_gc(seq) / length(seq)
     end
 end
 
+function count_gc(seq::Sequence)
```

Comment:
Should this have the same name as count_gc in operators.jl?

Reply:
Ohh i c. Its just for arbitrary other nucleic acid alphabets that don't fall into DNAAlphabet or RNAAlphabet... Cool.

Reply:
That's right. Thank you for reviewing it.

Reply:
Happy to review anytime!